src/main/scala/higherkindness/rules_scala/common/interrupt/BUILD

```diff
@@ -0,0 +1,14 @@
+load("//rules:scala.bzl", "scala_library")
+load("//rules:scalafmt.bzl", "scala_format_test")
+
+scala_library(
+    name = "interrupt",
+    srcs = glob(["*.scala"]),
+    scala = "//src/main/scala:bootstrap",
+    visibility = ["//visibility:public"],
```

Comment:
Yep. Will do.

Reply:
Actually, not sure I can do this. I need this to be public because there's workers that lives outside the ruleset that need access to InterruptUtil, otherwise they need to write their own version of it. Think the twirl compiler and play routes.

Reply:
> Think the twirl compiler and play routes.
Ohhh, I didn't realize those were implemented as workers. Nevermind, then.